### ISSUE TITLE
OS X 10.11 El Captain / SASS problem

### ISSUE THREAD

**Issue by diim010** [oklai/koala#466]
thanks for wonderful tool, but i have a problem with my  Mac OS X 10.11.1 (15B42) El Captain and latest version of Koala. When i try compile Sass/Scss files Koala show error.
![2015-11-08 22 40 53](https://cloud.githubusercontent.com/assets/4925794/11022565/351be7f4-866a-11e5-95d2-30b5cba85696.png)
.

**Comment by lholsti** [comment]
I have the same issue. It seems to come from line 150 in SassCompiler.js. I made the error print the command executed (line 153:  self.throwError(stderr, command);) and ran it in the terminal without errors.

I think the error is the same as issue 162 (https://github.com/oklai/koala/issues/162). Changing parameters on line 150 to: 
{timeout: 20000, maxBuffer: 10000*1024}
seems to solve the problem.

I came across the problem when editing the css from a site template. The file in question has about 20,000 lines in total, including comments and empty lines.

Hope this helps.
